examples/python/linear_regression.py

```diff
@@ -41,6 +41,6 @@ def loss_fn(w):
 throughput = num_iters / (toc - tic)
 
 print(
-    f"Loss {loss.item():.5f}, Mean Squared Eror : |w-w*| = {error_norm:.5f}, "
+    f"Loss {loss.item():.5f}, L2 Distance (Norm of difference) : |w-w*| = {error_norm:.5f}, "
```

Comment:
```suggestion
    f"Loss {loss.item():.5f}, L2 distance: |w-w*| = {error_norm:.5f}, "
```

Let's keep it a little shorter, don't think both comments are needed